javascript-modal/main.js

```diff
@@ -0,0 +1,24 @@
+let count = 0;
```

Comment:
Instead of using a number here, I would use a boolean. Something like `let modalOpen = false`